views/landing_page.handlebars

```diff
@@ -1,21 +1,90 @@
-<main>
+<head>
+ <style>
+
+label {
+  padding: 12px 12px 12px 0;
+  display: inline-block;
+}
+.add_user{
+  border-radius: 5px;
+  background-color: #e6b2b2;
+  padding: 20px;
+  text-align: center;
+
+  box-shadow: 0 0.4px 0.4px rgba(128, 128, 128, 0.109),
+    0 1px 1px rgba(128, 128, 128, 0.155),
+    0 2.1px 2.1px rgba(128, 128, 128, 0.195),
+    0 4.4px 4.4px rgba(128, 128, 128, 0.241),
+    0 12px 12px rgba(128, 128, 128, 0.35);
+}
+.new_project_board{
+  border-radius: 5px;
+  background-color: rgb(214, 247, 187);
+  padding: 20px;
+  text-align: center;
+
+  box-shadow: 0 0.4px 0.4px rgba(128, 128, 128, 0.109),
+    0 1px 1px rgba(128, 128, 128, 0.155),
+    0 2.1px 2.1px rgba(128, 128, 128, 0.195),
+    0 4.4px 4.4px rgba(128, 128, 128, 0.241),
+    0 12px 12px rgba(128, 128, 128, 0.35);
+}
+.project_button{
+  border-radius: 5px;
+  background-color: rgb(247, 246, 187);
+  padding: 20px;
+  text-align: center;
+
+  box-shadow: 0 0.4px 0.4px rgba(128, 128, 128, 0.109),
+    0 1px 1px rgba(128, 128, 128, 0.155),
+    0 2.1px 2.1px rgba(128, 128, 128, 0.195),
+    0 4.4px 4.4px rgba(128, 128, 128, 0.241),
+    0 12px 12px rgba(128, 128, 128, 0.35);
+}
+
+
+}
+ </style>
+</head>
+<body>
+    <main>
     <div><h1>Welcome to Leyamo Kanban</h1></div>
+    
     <aricle>
-        <form action="/add_user" method="POST">
+        <form class="add_user" 
+          action="/add_user" method="POST">
```

Comment:
can you keep it on the same line as class="add_user" ?

Reply:
okay thank you I will do that.